source/includes/stackpath/_delivery_domains.md

```diff
@@ -117,4 +117,50 @@ Delete a delivery domain of a site in a given [environment](#administration-envi
 Attributes | &nbsp;
 ------- | -----------
 `taskId` <br/>*string* | The task id related to the site deletion.
+`taskStatus` <br/>*string* | The status of the operation.
+
+<!-------------------- CREATE A DELIVERY DOMAIN -------------------->
+
+### Create a delivery domain
+
+```shell
+curl -X POST \
+    -H "MC-Api-Key: your_api_key" \
+    -d "request_body" \
+    "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/deliverydomains?siteId=a2fefb15-7e31-4c72-87e0-5a892e91c8d9&operation=Create"
+```
+
+> Request body example:
+
+```json
+{
+  "domain": "new-domain.com",
+  "validatedAt": "2021-02-26T19:00:15.177411Z"
+}
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "57fc8d89-6f13-451b-8b66-fcd96e1fedbd",
+  "taskStatus": "SUCCESS"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/deliverydomains?siteId=:id&operation=Create</code>
+
+Create a delivery domain of a site in a given [environment](#administration-environments).
+
+Required | &nbsp;
+---------- | -----------
+`domain`<br/>*String* | The site's domain name.
+
+Optional | &nbsp;
+---------- | -----------
+`validatedAt`<br/>*String* | The date the domain was validated to be pointing to Stackpath.
```

Comment:
Is this required for the creation? The API does not document it

Reply:
This is a value returned and should not be provided

Reply:
removed https://github.com/cloudops/cloudmc-api-docs/pull/317/commits/5ecc01b84a7d8218080b49460ff1e72977b94125